Running-Mastodon/Production-guide.md

```diff
@@ -63,6 +63,12 @@ server {
     try_files $uri @proxy;
   }
 
+  location /assets {
+    # Serve static, checksummed assets directly from nginx as a perf boost
+    root /home/mastodon/live/public;
```

Comment:
note that this is duplicated from the try_files on line 63 and the root declaration on line 49. You probably only need the cache-control header here.